src/header/Header.js

```diff
@@ -74,8 +74,8 @@ class Header extends Component {
       barStyle,
       children,
       linearGradientProps,
-      ViewComponent = linearGradientProps && global.Expo
-        ? global.Expo.LinearGradient
+      ViewComponent = linearGradientProps
```

Comment:
I think this might be just slightly easier to read as:
```js
ViewComponent = (linearGradientProps || !backgroundImage) ? View : ImageBackground
```